src/NetAppFiles/NetAppFiles/Account/SetNetAppFilesAccount.cs

```diff
@@ -0,0 +1,90 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.NetAppFiles.Common;
+using Microsoft.Azure.Commands.NetAppFiles.Models;
+using Microsoft.Azure.Management.NetApp;
+using Microsoft.Azure.Management.NetApp.Models;
+using Microsoft.Azure.Commands.NetAppFiles.Helpers;
+using System.Collections.Generic;
+
+// Note:
+// Both set and Update need to exist
+// Patch of active directories can only alter the content
+// to remove the active directory a put is required
+
+namespace Microsoft.Azure.Commands.NetAppFiles.Account
+{
+    [Cmdlet(
+        "Set",
+        ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "NetAppFilesAccount",
+        SupportsShouldProcess = true,
+        DefaultParameterSetName = FieldsParameterSet), OutputType(typeof(PSNetAppFilesAccount))]
+    [Alias("Set-AnfAccount")]
+    public class SetAzureRmNetAppFilesAccount : AzureNetAppFilesCmdletBase
+    {
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The resource group of the ANF account")]
+        [ValidateNotNullOrEmpty]
+        [ResourceGroupCompleter()]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The location of the resource")]
+        [ValidateNotNullOrEmpty]
+        [LocationCompleter("Microsoft.NetApp/netAppAccounts")]
+        public string Location { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The name of the ANF account")]
+        [ValidateNotNullOrEmpty]
+        [Alias("AccountName")]
+        public string Name { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "A hashtable array which represents the active directories")]
+        [ValidateNotNullOrEmpty]
+        public Hashtable[] ActiveDirectories { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "A hashtable which represents resource tags")]
+        [ValidateNotNullOrEmpty]
+        [Alias("Tags")]
+        public Hashtable Tag { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            var netAppAccountBody = new NetAppAccount()
+            {
+                Location = Location,
+                ActiveDirectories = (ActiveDirectories != null) ? ModelExtensions.ConvertActiveDirectoriesFromPs(ActiveDirectories) : new List<ActiveDirectory>(),
+                Tags = Tag
+            };
+
+            if (ShouldProcess(Name, "Create the new account"))
```

Comment:
Provide details on the identity of the created account (resource id or account name)